pkg/autoscaler/autoscaler.go

```diff
@@ -337,10 +333,16 @@ func (a *Autoscaler) Scale(ctx context.Context, now time.Time) (int32, bool) {
 			a.panicTime = &now
 			a.maxPanicPods = desiredPanicPodCount
 		}
+		// Never scale to 0 in the panic window.
```

Comment:
Additionally, using `a.maxPanicPods` is to make sure we never scale down at all during the panic window.  To prevent flapping.  Can you roll that into your comment?